### PR TITLE
move top level nav items into TopLeft items component

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?

This PR moves `transactions`, `account`, `lang` from `NavBar` to mobile collapse and topleftItems. The Navigation will further be refactored to use the new `items=` API from menu. This is the first step of adding these NavItems and will separate concerns a bit from mobile and desktop headers.
### Mobile
CleanShot 2022-10-17 at 20 58 29@2x

### Desktop
CleanShot 2022-10-17 at 20 58 18@2x

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2233** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2233" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2233?utm_source=stack-comment).

File: src/components/Navbar/index.tsx
Comment (benschac) [inline src/components/Navbar/index.tsx:11]: should be using `DesktopNavigation` here.

File: src/components/Navbar/DeskTopNavigation.tsx
Comment (benschac) [inline src/components/Navbar/DeskTopNavigation.tsx:21]: this file needs to be moved to https://github.com/jbx-protocol/juice-interface/pull/2234/files


and `TopLeftItems` should be deleted since it's a rename.

**Review by jmilldotdev — APPROVED** [review]

**Comment by benschac** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2233?utm_source=stack-comment).


Job Reference: QYaAht6eMvR7hTnqghel

**Pull request merged**
